### PR TITLE
release-it- use bamper plugin, pyproject update poetry settings

### PR DESCRIPTION
**Pull request by oharan2**
- Verified VERSION file
- Updated project metadata and dependencies
- Updated .release-it file `change-log` and `plugins` sections
- Updated `poetry.lock` according new changes

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+)
+'''
+
+[tool.isort]
+line_length = 88
+profile = "black"
+
 [tool.poetry]
 name = "rosa-python-client"
 version = "1.0.28"
```
Comment (myakove) [inline pyproject.toml:21]: version should be 0.0.0
And you should have a VERSION file with 1.0.28

File: pyproject.toml [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
 
 [tool.poetry.dependencies]
 python = "^3.8"
 redhat-qe-cloud-tools = "^1.0.6"
 python-benedict = "^0.32.0"
 python-simple-logger = "^1.0.5"
 
-
 [tool.poetry.group.dev.dependencies]
 ipdb = "^0.13.13"
```
Comment (myakove) [inline pyproject.toml:41]: Add ipython

**Review by myakove — CHANGES_REQUESTED** [review]
Please update the PR title to be more informative and add more info in the first comment

File: pyproject.toml (hunk 2 above)
Comment (oharan2) [reply pyproject.toml:41]: Done

**Review by myakove — CHANGES_REQUESTED** [review]
PR title is not updated, update it

File `.release-it.json` should be change to handle the VERSION file

```
"plugins": {
    "@j-ulrich/release-it-regex-bumper": {
      "out": [
        {
          "files": [
            "pyproject.toml"
          ],
          "search": "version = \"\\d+\\.\\d+\\.\\d+\"",
          "replace": "version = \"{{version}}\""
        }
      ]
    }
```
should be replaced with 

```
"plugins": {
    "@release-it/bumper": {
      "in": "VERSION",
      "out": "VERSION"
    }
  }

```

please update `release-it` section in the README
`npm install --save-dev @release-it/bumper` should be execute outside the repository directory (I think ~/ is good place)

File: pyproject.toml [hunk 3]
```diff
@@ -1,21 +1,61 @@
+[tool.black]
+line-length = 88
+target_version = ['py37', 'py38', 'py39', 'py310', "py311"]
```
Comment (rnetser) [inline pyproject.toml:3]: please remove `py37`

**Comment by rnetser** [comment]
> Please update the PR title to be more informative and add more info in the first comment

not addressed

File: pyproject.toml [hunk 4]
```diff
@@ -1,21 +1,61 @@
+[tool.black]
```
Comment (rnetser) [inline pyproject.toml:1]: https://github.com/RedHatQE/rosa-python-client/pull/71#pullrequestreview-1558368463 is not addressed

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**